content/02.introduction.md

```diff
@@ -8,7 +8,8 @@ The first known cases were located in Wuhan City within the Hubei province of Ch
 At the time of the WHO's first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf], 282 confirmed cases had been identified, primarily in China, but also 1-2 exported cases had been identified in several neighboring countries (Thailand, Japan, and the Republic of Korea).
 One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf].
 On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
-On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf?sfvrsn=99251b2b_4].
+On April 4, 2020, the WHO reported that the number of confirmed cases had surpassed one million globally [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf].
+The number of cases continues to rise daily, with up-to-date numbers tracked by sources such as Google [@url:https://www.google.com/search?q=google+covid+counts].
```

Comment:
Note that this may create a merge conflict with #480, where I fixed the figure caption.  We can be careful with whichever of the two pull requests is merged second.